src/markdown/inline/parsers.ts

```diff
@@ -5,14 +5,18 @@ import type {IElement} from '../../html/types';
 import type {TTokenizer} from '../../types';
 import type * as types from './types';
 
-const REG_INLINE_CODE = /^(`+)\s*([\s\S]*?[^`])\s*\1(?!`)/;
+const REG_INLINE_CODE = /^(`+)\s*([\s\S]*?[^`])\s*\1(?!`)(?:\{(?:\.|:)([^}]+)\})?/;
```

Comment:
Limit size of language label:

```suggestion
const REG_INLINE_CODE = /^(`+)\s*([\s\S]*?[^`])\s*\1(?!`)(?:\{(?:\.|:)([^}]{1,32})\})?/;
```